frontend-react/src/hooks/UseSenderResource.ts

```diff
@@ -23,13 +30,14 @@ export const useSenderResource = () => {
             authorizedFetch,
         ]
     );
-    const { data, isLoading } = rsUseQuery(
+    const { data, isLoading, isInitialLoading } = rsUseQuery(
```

Comment:
Added `isInitialLoading` to check if the query is disabled or not.  I'd like to add `isEnabled` to our useQuery wrapper, but that would result in a lot of test files getting changed 😬